[REF] remove inactive members (2/2)

legalsylvain opened the pull request:
* Based on top of https://github.com/OCA/repo-maintainer-conf/pull/9. Please merge before the other one.

*  Remove people listed in the previous issue. except :
    * @CharlineDumontet (recent member of oca-consultants)
    * @Ylemoing (recent member of oca-consultants)
    * @jgrandguillaume removed from all the psc, except the board one

**Removal :** 

* [REM] Remove @YannickB. (no activity since June 2022)
* [REM] Remove @stephankeller. (no activity since February 2016)
* [REM] Remove @sebastiken. (no activity since September 2021)
* [REM] Remove @rgarnau. (no activity since March 2018)
* [REM] Remove @redarouichi. (no activity since January 2016)
* [REM] Remove @RBelfiore. (no activity since April 2022)
* [REM] Remove @plamarche. (no activity since April 2016)
* [REM] Remove @okuryan. (no activity since August 2022)
* [REM] Remove @nuria-opusvl. (no activity since October 2015)
* [REM] Remove @mmalorni. (no activity since March 2020)
* [REM] Remove @mathi123. (no activity since August 2022)
* [REM] Remove @lucmaurer. (no activity since April 2022)
* [REM] Remove @lauracvilla-zz. (no activity since February 2020)
* [REM] Remove @labaggio. (no activity since March 2021)
* [REM] Remove @kloss17. (no activity since March 2021)
* [REM] Remove @jgrandguillaume. (no activity since July 2022)
* [REM] Remove @ierrajai. (no activity since January 2016)
* [REM] Remove @gmader. (no activity since October 2017)
* [REM] Remove @Fouad-AGORA. (no activity since January 2016)
* [REM] Remove @FFernandez-PlanetaTIC. (no activity since March 2020)
* [REM] Remove @fblauer. (no activity since November 2022)
* [REM] Remove @EvvFoxy. (no activity since February 2022)
* [REM] Remove @eugen-don. (no activity since December 2017)
* [REM] Remove @Ehtaga. (no activity since July 2018)
* [REM] Remove @duanyp1991. (no activity since September 2018)
* [REM] Remove @ddico. (no activity since September 2022)
* [REM] Remove @DayssamAgora. (no activity since March 2016)
* [REM] Remove @coodec. (no activity since April 2017)
* [REM] Remove @carlosrve. (no activity since September 2021)
* [REM] Remove @Capriatto. (no activity since May 2022)
* [REM] Remove @albariera. (no activity since July 2022)

**Rational :** 

* there are many people defined as responsible of repo, that are no longer active since a lot of time. 
* For the analyis, I listed the people that did'nt do a single "activity" on github since at least a year. It means, no PR, no commit, no comment on any issue or PR.
* Having ghost representatives present a number of problems for me : 
    *  Theoritical security problems. It's quite annoying to have inactive people having merge / adminitration right on OCA repos.
    *  at the moment, it looks as if people are taking care of the repos, when they're not. Sometimes people are pinged but don't respond, which creates a bad experience for contributors.
    * Furthermore, if you clean up these inactive managers, you'll find that some repos have been completely abandoned. This may raise questions. (Ex: github.com/OCA/dotnet, vertical-community, l10n-morocco)
    * If there are PSC without members, maybe this will encourage new people to apply to take care of the repos.

**Important Note :** 
- Just because we're withdrawing names doesn't mean these people aren't otherwise involved in the OCA.
- if people want to reinvest in a functional area later on, that's perfectly possible. Especially since the creation of repo-maintainer-conf, it's very quick to do.

rvalyi commented:
BTW the same reasonnning can be applied to some module maintainers who are in fact not active maintainers at all, when other more active developpers could be promoted as maintainers. Probably harder to automate, but good to keep in mind when we see the opportunity to clean the mess, when porting modules for instance.

One year is probably a good deadline for people who were listed as repo maintainers/PSC. As for module maintainers may be a bit larger time window (18 months?) could be considered as people could easily have been too busy and miss occasions to maintain their modules.

simahawk commented:
@legalsylvain thanks for the proposal and the cleanup w/ specific commits :love_you_gesture:  
The 1st PR is merged, can you rebase?

okuryan commented:
@legalsylvain you can remove me from "local-estonia-maintainers" and "local-ireland-maintainers". But the rest, please, leave. Though I'm rarely doing myself smth there. But on other repositories where I attached to I do not think there will be anybody else reply to local chat messages. For example, right now I've being asked to review one of the PRs in some local chats that we have in CIS region.

legalsylvain commented:
> @legalsylvain you can remove me from "local-estonia-maintainers" and "local-ireland-maintainers". But the rest, please, leave. Though I'm rarely doing myself smth there. But on other repositories where I attached to I do not think there will be anybody else reply to local chat messages. For example, right now I've being asked to review one of the PRs in some local chats that we have in CIS region.

OK Done. 
- removed from estonia / ireland. 
- still in poland / russia / belarus.


> @legalsylvain thanks for the proposal and the cleanup w/ specific commits 🤟 The 1st PR is merged, can you rebase?

rebased.

tafaRU reviewed (approved):
Thanks! :+1:

simahawk commented:
Let's see if it works...

/ocabot rebase

simahawk commented:
@legalsylvain cannot be merged or rebased: can you have a look pls?

legalsylvain commented:
> @legalsylvain cannot be merged or rebased: can you have a look pls?

yes, #12 conflicted this PR.

PR Rebased, could be great now.
